doc/source/serve/doc_code/load_shedding.py

```diff
@@ -0,0 +1,51 @@
+# flake8: noqa
+# fmt: off
+
+# __example_deployment_start__
+import time
+from ray import serve
+from starlette.requests import Request
+
+@serve.deployment(
+    # Each replica will be sent 2 requests at a time.
+    max_concurrent_queries=2,
+    # Each caller will queue up to 2 requests at a time
+    # (beyond those that are sent to replicas).
```

Comment:
```suggestion
    # (beyond those that Serve sends to replicas).
```

Reply:
Not sure if Serve is the sender, please correct accordingly. Just trying to avoid the passive voice.